### PR TITLE
update: enum error accuracy

### PR DESCRIPTION
**Pull request by sunrabbit123**
I will work next situation

```ts
enum T1 {
    a = "1",
    e = "a" + 1  // error
}
```

### CODE REVIEW

File: crates/stc_ts_file_analyzer/src/analyzer/enums.rs [hunk 1]
```diff
@@ -655,3 +671,14 @@ impl Visit<RExpr> for LitValidator<'_> {
         }
     }
 }
+
+fn has_not_str_and_not_plus(bin: &RBinExpr) -> bool {
```
Comment (kdy1) [inline crates/stc_ts_file_analyzer/src/analyzer/enums.rs:675]: Can you rename this to `non_str_nor_plus`?

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thank you!

File: crates/stc_ts_file_analyzer/src/analyzer/enums.rs (hunk 1 above)
Comment (sunrabbit123) [reply crates/stc_ts_file_analyzer/src/analyzer/enums.rs]: fixed it

**Review by kdy1 — APPROVED** [review]
Thanks!

**Pull request merged**